src/service/http/nebula/c/charts2.js

```diff
@@ -344,11 +344,28 @@ export class Charts {
 
         // if data specified as pair [], it will be used as min/max display for each label
         // 'bar'=>column, 'horizontalBar' for normal bar
-        this.displayBar = (chartId, json, keys, metrics, vertical) => {
+        this.displayBar = (chartId, json, keys, metrics, vertical, multiple = false) => {
```

Comment:
eventually one day we will need to refactor these displayXxx function to accept list to display collection as generic, fine for now.